modules/deploy/partials/cloud/controlplane-api.adoc

```diff
@@ -1,14 +1,14 @@
-= Use the Control Plane API
-:description: Use the Control Plane API to manage resources in your Redpanda Cloud organization.
-:page-cloud: true
-:page-beta: true
 :tag-clusters: api:ROOT:cloud-api.adoc#tag--ClusterService
 :tag-networks: api:ROOT:cloud-api.adoc#tag--NetworkService
 :tag-operations: api:ROOT:cloud-api.adoc#tag--OperationService
 :tag-resource-groups: api:ROOT:cloud-api.adoc#tag--ResourceGroupService
+:tag-serverless-regions: api:ROOT:cloud-api.adoc#tag--ServerlessRegionService
+:tag-serverless-clusters: api:ROOT:cloud-api.adoc#tag--ServerlessClusterService
 
 The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. The Control Plane API enables you to programmatically manage your organization's cloud environment outside of the Redpanda Cloud UI. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
```

Comment:
Link to the Terraform provider / content, please.

Majority of serverless users will use the programmatic API via Terraform.